tasks/github_tasks.py

```diff
@@ -270,3 +279,34 @@ def assign_team_label(_, pr_id=-1):
                 print(label_name, 'label assigned to the pull request')
             except github.GithubException.GithubException:
                 print(f'Failed to assign label {label_name}')
+
+
+@task
+def notify_community_pr(_, repo='', pr_id=-1):
+    import slack
+
+    # Get review teams / channels
+    gh = GithubAPI()
+
+    # Find teams corresponding to file changes
+    teams = _get_teams(gh.get_pr_files(pr_id)) or ['@datadog/agent-all']
+    channels = [GITHUB_SLACK_MAP[team.lower()] for team in teams if team if team.lower() in GITHUB_SLACK_MAP]
+
+    # Update labels
+    gh.add_pr_label(pr_id, 'community')
+    gh.add_pr_label(pr_id, 'team/triage')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe this should be global variable in the file, I'm not a fan of hardcoding label names into function, wdyt ?

Reply:
I agree, and same thing for `@datadog/agent-all` which is used several times